test/glific/flows/message_vars_parser_test.exs

```diff
@@ -311,6 +312,11 @@ defmodule Glific.Flows.MessageVarParserTest do
           "category" => "Civic ",
           "inserted_at" => "2022-02-21T13:07:11.722277Z",
           "input" => "2"
+        },
+        "storedanswer" => %{
+          "DetectedResponse" => "",
+          "OperationStatus" => "CONTINUED_ASSESSMENT",
+          "QuestionResponseID" => 1_083_000_000_113_341
```

Comment:
@pankaj-ag  didn't understand why this is needed

Reply:
I wrote few test cases which github was showing in the code coverage reports.